### PR TITLE
Updating login and sign up, to remove redundancy and correction for password strength.

### PR DESCRIPTION
**Pull request by amanex007**
IMPORTANT NOTES (please read, then delete):

* The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Introduce the first version of the collection editor.".

* Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables GitHub to link the PR to the corresponding bug.

Please also make sure to follow the [style rules](https://github.com/Cloud-CV/EvalAI/blob/master/.github/CONTRIBUTING.md#style-rules).

### CODE REVIEW

File: frontend/src/views/web/auth/login.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
                 <i ng-if="!canShowPassword" class="fa fa-eye pointer"></i>
                 <i ng-if="canShowPassword" class="fa fa-eye-slash pointer"></i>
             </span>
-            <label for="password">Password (min 8 characters) *</label>
+            <label for="password">Password *</label>
```
Comment (pushkalkatara) [inline frontend/src/views/web/auth/login.html]: Looks good.

File: frontend/src/views/web/auth/signup.html [hunk 2]
```diff
@@ -36,8 +36,10 @@
                 <i ng-if="canShowPassword" class="fa fa-eye-slash pointer"></i>
             </span>
             <label for="password">
-                <strong>Password-minlength:8 required</strong>
+                <strong>Password (minimum 8 Characters)</strong>
```
Comment (pushkalkatara) [inline frontend/src/views/web/auth/signup.html]: We can use a * here in the label.

**Comment by pushkalkatara** [comment]
The password strength is coming in next field. Please fix it. it should come in the same field of password.

**Comment by pushkalkatara** [comment]
Good work. Approving the task!

**Comment by amanex007** [comment]
Haha, thanks, nvr thought open source would be so much fun.

**Comment by pushkalkatara** [comment]
@amanex007 Please reopen the PR.

**Pull request closed without merging**